openedx/core/djangoapps/edx_global_analytics/tasks.py

```diff
@@ -30,20 +31,27 @@ def count_data():
     # Get IP address of the platform and convert it to latitude, longitude.
     ip_data = requests.get('http://freegeoip.net/json')
     ip_data_json = json.loads(ip_data.text)
-    platform_latitude, platform_longitude = olga_settings.get("PLATFORM_LATITUDE"), olga_settings.get("PLATFORM_LONGITUDE")
+
+    platform_latitude = olga_settings.get("PLATFORM_LATITUDE")
+    platform_longitude = olga_settings.get("PLATFORM_LONGITUDE")
    
     if platform_latitude and platform_longitude:
         latitude, longitude = platform_latitude, platform_longitude
     else:
         latitude, longitude = ip_data_json['latitude'], ip_data_json['longitude']
 
-    # Get students amount within current platform.
-    students_amount = UserProfile.objects.count()
+    # Get count of active students (if logged in during last 30 days)
+    # 30th day <= Current last login date <= Today
+    active_border = (datetime.datetime.now() - datetime.timedelta(days=30))
```

Comment:
What do you mean, do I need add `30` for `aws.py` (local) settings or `lms.env.json` (OPENEDX_LEARNERS_GLOBAL_ANALYTICS) settings?